cgi-bin/CATS/UI/Contests.pm

```diff
@@ -29,7 +29,7 @@ sub contests_new_frame {
     init_template($p, 'contests_new.html.tt');
 
     my $date = $dbh->selectrow_array(qq~
-        SELECT CURRENT_TIMESTAMP $FROM_DUMMY~);
+        SELECT CURRENT_TIMESTAMP $db->{FROM_DUMMY}~);
```

Comment:
How did it work before without import?

Reply:
I removed '$FROM_DUMMY' and added '$db' in previous commit
https://github.com/klenin/cats-main/pull/350/commits/3be5a35f79ee66dceb63d3e021431da6edfe2b81#diff-f11fd1fbf18b894256063c6ee9eb8f14R12